docs/content/troubleshooting/_index.md

```diff
@@ -1,4 +1,8 @@
 ---
 title: Troubleshooting
+description:
 weight: 2000
+menu:
```

Comment:
I don't think this menu metadata actually is being used at present, but it doesn't hurt anything to be here.

Reply:
Also, if you want these to be present for later use but not actually build (and therefore publish), add the following to the frontmatter:

```yaml
_build:
  render: never
  list: never
```